src/module.c

```diff
@@ -2254,6 +2254,29 @@ int moduleIsModuleCommand(void *module_handle, struct serverCommand *cmd) {
     return (cp->module == module_handle);
 }
 
+/* ValkeyModule_UpdateRuntimeArgs can be used to update the module argument values.
+ * The function parameter 'argc' indicates the number of updated arguments, and 'argv'
+ * represents the values of the updated arguments.
+ * Once 'CONFIG REWRITE' command is called, the updated argument values can be saved into conf file.
+ *
+ * Returns:
+ * - VALKEYMODULE_ERR on failure.
+ */
```

Comment:
I think this function can't return VALKEYMODULE_ERR. I can't see any return like this.

But it always returns VALKEYMODULE_OK. It was not mentioned.

Reply:
The comment is addressed